source/docs/hardware/sensors/encoders-hardware.rst

```diff
@@ -27,25 +27,32 @@ These encoders vary in how they are mounted to the mechanism in question.  In ad
 Shafted encoders
 ^^^^^^^^^^^^^^^^
 
-.. todo:: add picture
+.. image:: images/encoders-hardware/grayhill-63r.jpg
```

Comment:
Is it possible to have a SVG picture here, it's a bit jarring. If not, it's fine.

Reply:
Added in 8330140.